VersionControlUI/src/commands/CDiff.cpp

```diff
@@ -111,6 +111,24 @@ Interaction::CommandResult* CDiff::execute(Visualization::Item*, Visualization::
 	return new Interaction::CommandResult{};
 }
 
+QString CDiff::descriptionForCommits(QString token, QList<QPair<QString, QString>> commits)
```

Comment:
pass this list as a `const &`. In general, pass QT classes (QList, QString, etc) by a const reference if you don't need to modify them.